README.md

```diff
@@ -1,7 +1,22 @@
 # **Observability Remote Write Proxy**
 
-### A proxy to authenticate remote write requests before they are forwarded to Observatorium. 
+### A proxy to accept unauthenticated remote write requests and forward them to Observatorium. Before forwarding the request, the proxy authenticates with the provided oidc credentials and adds the obtained token to the request. The proxy is intended to be used by customers who want to send data to Observatorium but don't want to store Observatorium credentials on their clusters.
+
+## **How to set up the Proxy locally**
+
+#### Flags that are needed to proxy to another prometheus instance:
+--proxy.forwardUrl=http://localhost:9091/api/v1/write
```

Comment:
I'd replace the localhost URL here with a placeholder like <Prometheus remote write URL>